go-controller/pkg/network-attach-def-controller/network_attach_def_controller.go

```diff
@@ -225,12 +226,13 @@ func (nadController *netAttachDefinitionController) sync(key string) error {
 	}
 }
 
-func (nadController *netAttachDefinitionController) handleErr(err error, key interface{}) {
+func (nadController *NetAttachDefinitionController) handleErr(err error, key interface{}) {
 	ns, name, keyErr := cache.SplitMetaNamespaceKey(key.(string))
 	if keyErr != nil {
 		klog.ErrorS(err, "Failed to split meta namespace cache key", "key", key)
 	}
-	if err == nil {
+
+	if err == nil || util.IsNetworkControllerTopologyNotSupportedError(err) {
```

Comment:
If would consider logging:

```
if util.IsNetworkControllerTopologyNotSupportedError(err) {
    klog.Info(...)
    err = nil
}
```

providing specific information that this is about the cluster network controller and not the normal network controller.